tests/integration/redirect/dts/tsconfig.json

```
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "strict": true,
+    "skipLibCheck": true,
+    "paths": {
+      "@src/*": ["./src/*"],
+      "rslog": ["./compile/rslog"]
+    }
```

```suggestion
    "paths": {
      "*": ["./src/*"],
      "@src/*": ["./src/*"],
      "rslog": ["./compile/rslog"]
    }
```

This is the only additional path alias I've had issues with. Looks like it still handles correctly without changes, so this looks good to me.

---

Great test case, I have added it to our tests.